test/integration/bun-types/fixture/test.ts

```diff
@@ -335,6 +336,16 @@ expectType(spy.mock.calls).is<[message?: any, ...optionalParams: any[]][]>();
 jest.spyOn(console, "log");
 jest.fn(() => 123 as const);
 
+// vi (Vitest-compatible) API: simple type checks
+expectType<typeof jest.fn>(vi.fn);
+expectType<typeof spyOn>(vi.spyOn);
```

Comment:
These type checks are mostly useless because they are effectively testing "x == x". I would suggest instead writing out some actual "pretend" usage of these properties/functions themselves and not necessarily using the expectType helper.